### PR TITLE
campaign service add error handling

### PR DESCRIPTION
**Pull request by shan-57blocks**
1, Campaign service error handling. First throw the error message from backend then the general message.
2, Create error message variables

### CODE REVIEW

**Review by ljiatu — COMMENTED** [review]
Are we showing error messages if any of these errors occur?

**Comment by shan-57blocks** [comment]
> Are we showing error messages if any of these errors occur?

Currently no, will just show '--'. I'm think of it.

**Comment by shan-57blocks** [comment]
> Are we showing error messages if any of these errors occur?

Updated

File: packages/huma-shared/src/utils/const.ts [hunk 1]
```diff
@@ -3,10 +3,23 @@ export const EARejectReason = 'Your wallet does not meet qualifications'
...
 export enum CURRENCY_CODE {
   USD = 840,
 }
 
 export const CAMPAIGN_REFERENCE_CODE = 'CAMPAIGN_REFERENCE_CODE'
 
 export const BP_FACTOR_NUMBER = 10000
+
+export enum HUMA_ACCOUNT_EXCEPTION {
+  IdTokenNotFoundException = 'IdTokenNotFoundException',
```
Comment (ljiatu) [inline packages/huma-shared/src/utils/const.ts:18]: Does the point service still use ID tokens?

File: packages/huma-widget/src/components/Lend/solanaSupply/8-PointsEarned.tsx [hunk 2]
```diff
@@ -86,14 +87,18 @@ export function PointsEarned({
...
-        const ownership = await CampaignService.checkWalletOwnership(
-          account,
-          networkType,
-          isDev,
-        )
-        setWalletOwnership(ownership)
-        if (!ownership) {
-          setAuthError('WalletNotSignedInException')
+        try {
+          const ownership = await CampaignService.checkWalletOwnership(
```
Comment (ljiatu) [inline packages/huma-widget/src/components/Lend/solanaSupply/8-PointsEarned.tsx:91]: Why is the campaign service still checking wallet ownership?

Comment (shan-57blocks) [reply packages/huma-widget/src/components/Lend/solanaSupply/8-PointsEarned.tsx]: Removed

File: packages/huma-shared/src/utils/const.ts (hunk 1 above)
Comment (shan-57blocks) [reply packages/huma-shared/src/utils/const.ts]: Updated. @ljiatu does identity service returns WalletMismatchException?

Comment (ljiatu) [reply packages/huma-shared/src/utils/const.ts]: No it doesn't.

**Review by ljiatu — APPROVED** [review]

File: packages/huma-shared/src/utils/const.ts (hunk 1 above)
Comment (shan-57blocks) [reply packages/huma-shared/src/utils/const.ts]: Removed

**Pull request merged**